packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -38,6 +38,9 @@ const Duration _kTransitionDuration = Duration(milliseconds: 167);
 const Curve _kTransitionCurve = Curves.fastOutSlowIn;
 const double _kFinalLabelScale = 0.75;
 
+/// Callback for building the error widget.
```

Comment:
Signature for a callback that builds an error widget.

See also:

  * [InputDecorator.errorBuilder], which is of this type, and passes the `errorText` given by [TextFormField.validator].

Reply:
Thanks for the suggestions. I improved the doc comments.